src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -399,7 +399,7 @@ typedef struct {
     /* Process management and PMI globals */
     int pname_set;
     int pname_len;
-    char addrname[MPIDI_OFI_MAX_NICS][FI_NAME_MAX];
+    char addrname[FI_NAME_MAX];
```

Comment:
Do we need to account for the new num_nics here too?

Reply:
@tarudoodi I don't think so. This change is in its own commit. `addrname` is currently used as a singular char array/string, and I think it was defined as a 2-d array by mistake.